includes/abstracts/class-wpum-wp-db-table.php

```diff
@@ -0,0 +1,357 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-wpum-wp-db-table.php, but found vipgoci-phpcs-scan-UYaG1t.php (*WordPress.Files.FileName.InvalidClassFileName*).